response_operations_ui/controllers/message_controllers.py

```diff
@@ -61,12 +59,23 @@ def get_conversation_count(params):
         raise NoMessagesError
 
 
-def get_thread_list(params):
-    logger.info("Retrieving threads list")
+def _get_secure_message_threads_params(survey_id, business_id, conversation_tab):
+    """creates a params dictionary"""
+    params = {'survey': survey_id,
+              'is_closed': 'true' if conversation_tab == 'closed' else 'false',
+              'my_conversations': 'true' if conversation_tab == 'my messages' else 'false',
+              'new_respondent_conversations': 'true' if conversation_tab == 'initial' else 'false'}
+    if business_id:
```

Comment:
Would it be reasonable to just add the business_id even if it's empty?  I'm probably wrong, but this seems oddly verbose and that would tidy it up a bit

Reply:
Disagree , these are the params passed to secure message and it currently expects params to be populated if they are designated. This was repeated code called from two places , so the current one makes the code Dryer than it was

Reply:
yeah that's fair